### PR TITLE
removing unused image files

### PR DESCRIPTION
**Pull request by samgutentag**
these image files are not used in any docs pages and can be safely removed

### CODE REVIEW

File: static/img/5g-on-helium/mobile-header.png
Comment (jthiller) [inline static/img/5g-on-helium/mobile-header.png:1]: We might want this one for the MOBILE token page.

File: static/img/home/docsheader.png
Comment (jthiller) [inline static/img/home/docsheader.png:1]: Will want this one when we complete the migration and revert the home page.

File: static/img/validators/faucet.png
Comment (jthiller) [inline static/img/validators/faucet.png:1]: Might be worth keeping around.

**Review by jthiller — CHANGES_REQUESTED** [review]
We'll want to use a couple of these.

File: static/img/5g-on-helium/mobile-header.png
Comment (samgutentag) [reply static/img/5g-on-helium/mobile-header.png:1]: restored!

File: static/img/validators/faucet.png
Comment (samgutentag) [reply static/img/validators/faucet.png:1]: restored!

File: static/img/home/docsheader.png
Comment (samgutentag) [reply static/img/home/docsheader.png:1]: restored!

**Review by jthiller — APPROVED** [review]
Thx!

**Pull request merged**